### PR TITLE
Remove periods from list

### LINKED ISSUE #33902: Remove periods from list in managing-notifications-from-your-inbox#custom-filter-limitations
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox#custom-filter-limitations

### What part(s) of the article would you like to see updated?

> Custom filters do not currently support:
> 
> - Full text search in your inbox, including searching for pull request or issue titles.
> - Distinguishing between the `is:issue`, `is:pr,` and `is:pull-request` query filters. These queries will return both issues and pull requests.
> - Creating more than 15 custom filters.
> - Changing the default filters or their order.
> - Search [exclusion](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#exclude-certain-results) using `NOT` or `-QUALIFIER`.

Remove periods from this list, as each line is not a complete sentence

### Additional information

Style guide [reference](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#lists)

### PR DESCRIPTION
**Pull request by SeanAverS**
### Why:
Remove periods from list as list items are not full sentences.

Closes: #33902 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

- Remove period from list items on lines 76-80

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by janusman582 — COMMENTED** [review]
Ok

File: content/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox.md
Comment (janusman582) [inline content/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox.md:1]: content/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox.md

**Review by janusman582 — COMMENTED** [review]
👈

**Review by nguyenalex836 — DISMISSED** [review]

**Review by nguyenalex836 — DISMISSED** [review]

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@SeanAverS Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**